GameData/RP-0/Avionics.cfg

```diff
@@ -921,14 +921,14 @@
 }
 
 //  Centaur astrionics.
-//  Approximates the maximum current Centaur + payload combined mass (23t + 25t).
+//  Centaur guidance provides control for both Centaur and Atlas stages
 
-@PART[RP0probeAvionics3-05m]:FOR[RP-0]
+@PART[RP0probeAvionics3-05m|RSBtankAtlasCentaur]:FOR[RP-0]
 {
 	MODULE
 	{
 		name = ModuleAvionics
-		massLimit = 45.0
+		massLimit = 600
```

Comment:
That's actually a significant change for the existing RP-1 avionics part. I wonder if it would be better if RSB config would be separate. Or how do those 2 parts compare to each other mass-wise?

Reply:
The dry mass of the RSB model with one engine and including the 600t guidance capability is 2405 kg - slightly less than the 2245-ish it weighs according to [the Atlas V user guide from ULA](https://www.ulalaunch.com/docs/default-source/rockets/atlasvusersguide2010.pdf), but still within the range.

The Saturn V RP-1 avionics part controls 3000t with a mass of 2t, or 1.5t per kg. Scaling this down to a part mass of 300kg of the existing RP-1 Centaur avionics would mean controlling 450t. I don't think its unreasonable to assume that the difference between technology which flew in the 1970's vs the 2000's could make up for that gap.

Reply:
My issue is that you're also changing a part called `RP0probeAvionics3-05m` which lies in the `Improved Avionics (1962-1963)` tech node. So if your part is unlocked at a later node then it shouldn't change the config for `RP0probeAvionics3-05m`.

Reply:
Didn't notice that - I'll separate them into two different configs.